scripts/automatic_tag.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/config"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/transport/ssh"
+	"golang.org/x/mod/semver"
+)
+
+const (
+	sdkRepo      = "git@github.com:stackitcloud/stackit-sdk-go.git"
+	patch        = "patch"
+	minor        = "minor"
+	coreOnlyFlag = "--core-only"
```

Comment:
The name `core-only` indicates that if this flag is not passed it will create tags for all services *and* core, but that's not the case, right?

Maybe we could have instead a required argument `target` that would wither be `core` or `all-services`, as the use case is generally creating a new tag for core *or* for services. This would also be easier to extend in case we want to allow creation of tags for just a specific service

Reply:
This is absracted by the make command, but still I think it could be a nice improvement

Reply:
good suggestion, done